added in fourth item in feature box and adjusted mobile responsiveness

chanonroy opened the pull request:
<img width="1214" alt="screen shot 2017-01-03 at 5 48 44 pm" src="https://cloud.githubusercontent.com/assets/16366896/21626060/fb05edde-d1dc-11e6-8980-1a3573b50b00.png">

Looking to contribute more, holla if you need more help

contra commented:
Copy is duplicated between this and the first item. You can revert the copy for the first item to what it used to be to rectify that.

chanonroy commented:
Ah. Just noticed now. 

**Re: first item's old copy**
Is this the copy you're referring to? (from the [marvelapp](https://marvelapp.com/gj54hh/screen/14587285) mockup)
`"By preferring code over configuration, gulp keeps things simple and makes complex tasks manageable"`

Looking at the overall picture now, I can see why the 'Easy to Learn' feature item may have been omitted. Let me know if this is still a worthwhile effort. Otherwise, I can pick up on an official 'help wanted' issue.

contra commented:
@chanonroy Yep, that copy. Title too. I think they're different enough to have both.

chanonroy commented on css/main.css:
```diff
@@ -66,6 +66,7 @@ body {
 }
 .subheading + .paragraph {
     margin-top: 10px;
+    padding: 0 30px 0 30px;
```
(optional) - makes padding for text copy look similar to the mockup

contra commented:
@chanonroy  It seems like the latest changes updated the existing copy to the old stuff, but didn't add the new section to replace the lost copy?

chanonroy commented:
@contra - it didn't carry over from the previous PR commit made earlier? On my fork the new section with lost copy is still there. Did it have to do with the way that the latest commit updated the PR?

contra commented:
Hmm, looks a little funky:

![image](https://cloud.githubusercontent.com/assets/425716/21630186/f8ec78e0-d1fc-11e6-9324-08d9c1ca7fb0.png)

chanonroy commented:
Hmm. It should not look like that.

chanonroy commented:
@contra - this is using the updated main.css, yes?

contra commented:
@chanonroy I merged your whole PR so all the changes in here were deployed.

chanonroy commented:
![commit](https://cloud.githubusercontent.com/assets/16366896/21630557/69322930-d200-11e6-9174-57b0723d301b.png)

I'm looking at the merge. I could be wrong, but I feel like every 'remove' line in red (#1) is something that I added, and that every 'add' line in green (#2) is something that I removed.

Something funky going on here.

chanonroy commented:
Worse comes to worse, we can always revert it to the original with 3 feature boxes. Lots of effort for very minimal changes.

contra commented:
@chanonroy Reverted it while you work it out.
